styles.css

```
@@ -8136,6 +8136,96 @@ body:has(#side-panel.active) #close-fullscreen-cover-btn {
+
+.eq-spectrum-pill,
+.eq-spectrum-toggle-in-graph {
+    display: inline-flex;
+    align-items: center;
+    gap: 6px;
+    padding: 4px 8px 4px 6px;
+    font-size: 0.7rem;
+    font-weight: 600;
+    letter-spacing: 0.02em;
+    text-transform: uppercase;
+    border: 1px solid rgba(255, 255, 255, 0.18);
```

Stylelint (17.6.0)

[error] Expected "rgba" to be "rgb" (color-function-alias-notation)

(color-function-alias-notation)